### DISCUSSION TITLE
Issue applying for student benefits

### DISCUSSION THREAD

**Discussion by Michaelihc** [community/community#128212] [GitHub Education] [answered]
### Hi everyone,

I kept receiving a location error when applying for student benefits. My teacher's submission has already been approved, and I check, we have the exact same school information, and we were on the same network. Any ideas? 

Error: "You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket."

**Comment by ItsLasa** [comment, marked as answer]
hey  Michaelihc ...

It sounds like you're running into a location verification issue while applying for student benefits on GitHub. Here are some steps to troubleshoot and potentially resolve this problem:

1.Enable Location Services:

Windows: Go to Settings > Privacy > Location and ensure that location services are turned on for your browser.
macOS: Go to System Preferences > Security & Privacy > Privacy > Location Services, and make sure your browser is checked.

2.Browser Location Permissions:

Chrome: Go to Settings > Privacy and security > Site Settings > Location and ensure that https://github.com is allowed to access your location.
Firefox: Go to Preferences > Privacy & Security > Permissions > Location and make sure https://github.com is allowed.
Edge: Go to Settings > Site permissions > Location and check that https://github.com is allowed.

3.Disable VPN:
Ensure that you're not connected to a VPN, as this can interfere with accurate location detection.
Verify Your Academic Affiliation
Documentation:

Make sure your academic affiliation documentation explicitly states if you are a remote student if applicable.
School Information:

Double-check the information you entered for your school to ensure it matches exactly with what your teacher submitted, including the spelling and formatting of the school name and address.
Clear Browser Data
Sometimes clearing your browser's cache and cookies can help resolve location issues. Go to your browser settings and clear your browsing data.
Submit a Support Ticket
If you've checked all the above and still encounter issues, you might need to contact GitHub Education support directly. Here's how:

Go to the [GitHub Education Support](https://education.github.com/discount_requests/new).
Select the option "My selected school has incorrect or incomplete information e.g. domains or campus location."
Provide detailed information about your issue, including:
The error message you received.
Confirmation that you are not using a VPN.
Screenshots of your location settings (if possible).
Your teacher's approved application details for reference.

Different Network: Try applying from a different network (e.g., your home Wi-Fi instead of campus Wi-Fi) to see if it changes the outcome.
Geolocation Test: Use an online tool to check if your browser correctly detects your location (e.g., [My Location](https://mylocation.org/)).

Hopefully, one of these solutions helps resolve the issue. Good luck!

**Reply by Michaelihc** [reply]
I got it, thanks

**Reply by samus-aran** [reply]
Really detailed response @ItsLasa Well done 🎉

**Reply by dhruv-cpi** [reply]
Hi GitHub Team,

I have applied for the GitHub Student Developer Pack but am facing some technical issues. Initially, my application was rejected due to issues with 2FA and billing details. After completing all the required guidelines, I reapplied and my application was approved. However, I am still unable to access the program as it shows a "coming soon" message.

Can you please help me gain access to the GitHub Student Developer Pack?

Thank you for your assistance.

Best regards,
Dhruv Sanchaniya

![Screenshot (195).png](https://github.zendesk.com/attachments/token/85tKI2ERFtnki6sBwMl9QdPw5/?name=Screenshot+%28195%29.png)
